### DISCUSSION TITLE
如何针对节点去限制最大最小宽高，现在transform的resizing只能全局限制

### DISCUSSION THREAD

**Discussion by Ardanas** [antvis/X6#4443] [Q&A]
![image](https://github.com/user-attachments/assets/051ae19a-0fff-49c7-816d-180aecbf43ed)
改配置只能全局限制

除了监听改变后再改变回来，还有其它变化吗，从源头限制

**Comment by Yeti-xxx** [comment]
可以修改大小拖拽工具，将最大宽度和高度设置在节点内的 /data 中，工具调用时读取maxWidth maxHeight，到达阈值时就不做对应的高度或宽度变化

**Reply by Ardanas** [reply]
大小拖拽工具指的是？@antv/x6-plugin-transform吗？
能具体说说吗，感谢

**Reply by Yeti-xxx** [reply]
@antv/x6-plugin-transform   是的 可以尝试自己写一个节点工具或插件，插件的话官方没有相关文档 需要你查看插件的代码

**Reply by Ardanas** [reply]
得咧，我去看下插件源码
